src/main/java/com/dreammaster/bartworksHandler/BioItemLoader.java

```diff
@@ -49,105 +52,74 @@ public class BioItemLoader {
+                .eut(TierEU.RECIPE_HV).addTo(sCentrifugeRecipes);
+
+        RA.stdBuilder().itemInputs(GT_Utility.getIntegratedCircuit(11), new ItemStack(BIOTEMS, 1, 1))
+                .itemOutputs(new ItemStack(BIOTEMS, 1, 2)).fluidInputs(GT_ModHandler.getDistilledWater(1000))
+                .noFluidOutputs().duration(30 * SECONDS).eut(TierEU.RECIPE_HV).addTo(sMixerRecipes);
+
+        GT_Values.RA.stdBuilder().itemInputs(Materials.MeatRaw.getDust(1)).noItemOutputs().noFluidInputs()
+                .fluidOutputs(new FluidStack(BIOFLUIDS[1], 125)).duration(15 * SECONDS).eut(TierEU.RECIPE_MV)
+                .addTo(sFluidExtractionRecipes);
+
+        GT_Values.RA.stdBuilder()
+                .itemInputs(
+                        new ItemStack(BIOTEMS, 8, 2),
+                        ItemList.Circuit_Chip_Stemcell.get(16),
+                        Materials.Salt.getDust(64))
+                .noItemOutputs()
+                .fluidInputs(
+                        FluidRegistry.getFluidStack("unknowwater", 4000),
+                        Materials.PhthalicAcid.getFluid(3000),
+                        new FluidStack(BIOFLUIDS[1], 1000))
+                .fluidOutputs(new FluidStack(BIOFLUIDS[2], 8000)).duration(60 * SECONDS).eut(TierEU.RECIPE_UV)
+                .addTo(sMultiblockChemicalRecipes);
+
         addBacterialVatRecipe(
                 new ItemStack[] { ItemList.IC2_Energium_Dust.get(8), Materials.Mytryl.getDust(1),
                         GT_ModHandler.getModItem(PamsHarvestCraft.ID, "seaweedItem", 64) },
                 CultureSet.get("TcetiEBac"),
                 new FluidStack[] { new FluidStack(BIOFLUIDS[2], 50) },
                 new FluidStack[] { new FluidStack(BIOFLUIDS[3], 50) },
                 1200,
-                BW_Util.getMachineVoltageFromTier(8),
+                (int) TierEU.RECIPE_UV,
                 100,
                 8,
                 0,
                 false);
         for (int i = 0; i < OreDictionary.getOres("cropTcetiESeaweed").size(); i++) {
-            RA.addCentrifugeRecipe(
-                    GT_Utility.getIntegratedCircuit(i + 1),
-                    NI,
-                    new FluidStack(BIOFLUIDS[3], 1000),
-                    NF,
-                    OreDictionary.getOres("cropTcetiESeaweed").get(i).copy().splitStack(64),
-                    NI,
-                    NI,
-                    NI,
-                    NI,
-                    NI,
-                    null,
-                    40,
-                    BW_Util.getMachineVoltageFromTier(8));
+            GT_Values.RA.stdBuilder().itemInputs(GT_Utility.getIntegratedCircuit(i + 1))
+                    .itemOutputs(OreDictionary.getOres("cropTcetiESeaweed").get(i).copy().splitStack(64))
+                    .fluidInputs(new FluidStack(BIOFLUIDS[3], 1000)).noFluidOutputs().duration(2 * SECONDS)
+                    .eut(TierEU.RECIPE_UV).addTo(sCentrifugeRecipes);
```

Comment:
Should add `.noOptimize()`

![image](https://github.com/GTNewHorizons/NewHorizonsCoreMod/assets/53441451/a1d88aab-cfc4-4351-9ee8-b5dcbd543d31)